swagger/swagger.yaml

```diff
@@ -133,6 +132,47 @@ paths:
           description: "Successful data fetch of rackmount catalogs"
       x-tags:
       - tag: "default"
+
+  /pollers:
```

Comment:
It looks like the content is described in responses description'"Successful data fetch of catalogs by class id"' so maybe it don't need to add new /pollers, just leverage /catalog API.  I thinks there is no poller here, just it's named by RackHD's /poller. so there's no direct relationship here

Reply:
/pollers and /catalog will get different data from UCSM.
Most data of /pollers are device status, like cpuStats while /catalog is usually device static information.

Reply:
Thanks for clarification